app/Console/Commands/InstallCommand.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class InstallCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'council:install';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Simplify installation process';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $this->intro();
+
+        if (!file_exists('.env')) {
+            exec('mv .env.example .env');
```

Comment:
Maybe we can use Filesystem to copy this instead of `exec`ing...

Reply:
nice idea, I just went and adapted the instructions on the Readme file for now, but there's much room for improvement and optimization :)